core/src/main/java/org/apache/metamodel/query/SelectItem.java

```diff
@@ -152,7 +166,24 @@ public SelectItem(Column column, FromItem fromItem) {
      * @param fromItem
      */
     public SelectItem(FunctionType function, Column column, FromItem fromItem) {
-        this(column, fromItem, function, null, null, null, false);
+        this(column, fromItem, function, null, null, null, null, false);
```

Comment:
It's not possible in Java to put anything before the call to ```this(...)``` so that's not possible.